opencti-platform/opencti-front/src/private/components/Error.jsx

```diff
@@ -86,7 +86,7 @@ export const BoundaryRoute = (props) => {
 };
 
 BoundaryRoute.propTypes = {
-  component: PropTypes.func,
+  component: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
this one is to fix the warnings in console due to the use of `lazy(...)` introduced for vite.